React/Views/RCTRefreshControl.m

```diff
@@ -52,18 +57,9 @@ - (void)beginRefreshing
 {
   // When using begin refreshing we need to adjust the ScrollView content offset manually.
   UIScrollView *scrollView = (UIScrollView *)self.superview;
-  CGPoint offset = {scrollView.contentOffset.x, scrollView.contentOffset.y - self.frame.size.height};
-
-  // `beginRefreshing` must be called after the animation is done. This is why it is impossible
```

Comment:
> In fact, it looks even better because the refreshControl is displayed without delay.

Do you mean there is no more animation or the animation is just faster / better now? This code was written on iOS 8 or 9 so it's possible it is not needed anymore and I think it's reasonable to remove it.